servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/SpliceFlatStreamToMetaSingleTckTest.java

```diff
@@ -1,45 +0,0 @@
-/*
- * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
```

Comment:
Can we adapt and move the tests to servicetalk-concurrent-api? Generally speaking, we try to have tck and unit tests for all operators.

Reply:
Right. This was not meant to be deleted int he final draft...